hbase-migration-tools/bigtable-hbase-replication/bigtable-hbase-replication-core/src/main/java/com/google/cloud/bigtable/hbase/replication/configuration/HBaseToCloudBigtableReplicationConfiguration.java

```diff
@@ -71,4 +71,22 @@ private HBaseToCloudBigtableReplicationConfiguration() {}
   // batch capacity by counts and bytes. These capacity are set by `replication.source.nb.capacity`
   // and `replication.source.size.capacity` config keys.
   public static final long DEFAULT_BATCH_SIZE_IN_BYTES = 500_000;
+
+  /**
+   * Enables logic to prevent replication loops from forming when this library is used in
+   * conjunction with a Bigtable-HBase replicator.
+   *
+   * <p>CBT and HBase qualifiers must match those used in CBT replicator configs for this mode to
+   * work.
+   */
+  public static final String ENABLE_TWO_WAY_REPLICATION_MODE_KEY =
+      "google.bigtable.replication.enable_two_way_replication";
+
+  public static final String SOURCE_HBASE_QUALIFIER_KEY =
+      "google.bigtable.replication.hbase_qualifier";
+  public static final String SOURCE_CBT_QUALIFIER_KEY = "google.bigtable.replication.cbt_qualifier";
+
+  public static final boolean DEFAULT_TWO_WAY_REPLICATION_MODE = false;
+  public static final String DEFAULT_SOURCE_HBASE_QUALIFIER = "SOURCE_HBASE";
```

Comment:
use some namespacing, keep it short but make it so that it doesn't collide with customer columns? CBT_REPL_HBASE / CBT_REPL_CBT ?